Makefile.tools

```diff
@@ -194,16 +197,40 @@ $(SHELLCHECK) shellcheck: $(TOOLS_DIR)
 		set -ex ;\
 		[[ -f $(SHELLCHECK) ]] && exit 0 ;\
 		cd $$(mktemp -d) ;\
-		curl -sSLo shellcheck-stable.tar.xz "https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.linux.x86_64.tar.xz";\
+		curl -sSLo shellcheck-stable.tar.xz "https://github.com/koalaman/shellcheck/releases/download/$(SHELLCHECK_VERSION)/shellcheck-$(SHELLCHECK_VERSION).linux.x86_64.tar.xz" ;\
 		tar -xJf shellcheck-stable.tar.xz ;\
-		cp shellcheck-stable/shellcheck $(SHELLCHECK) ;\
+		cp shellcheck-$(SHELLCHECK_VERSION)/shellcheck $(SHELLCHECK) ;\
 		version=$(SHELLCHECK_VERSION) ;\
 		$(SHELLCHECK) -V | grep -q $${version##v} ;\
 	}
 
+.PHONY: yq
+$(YQ) yq: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		[[ -f  $(YQ) ]] && exit 0 ;\
+                cd $$(mktemp -d) ;\
+		wget https://github.com/mikefarah/yq/releases/download/$(YQ_VERSION)/yq_linux_amd64 -O $(TOOLS_DIR)/yq ;\
+		chmod +x $(TOOLS_DIR)/yq ;\
+	}
+
 .PHONY: jsonnet-tools
 jsonnet-tools: jsonnet jsonnetfmt jsonnet-lint jb gojsontoyaml
 
+.PHONY: jq
+jq: 
+	@{ \
+		set -ex ;\
+		sudo apt-get install jq ;\
```

Comment:
We keep the tool installs distro agnostic. and apt install won't work on my machine. Let's either install this like the other tools or add a gate that checks for presence and errors out if it's not in the PATH